android/app/build.gradle

```diff
@@ -104,12 +220,4 @@ task copyDownloadableDepsToLibs(type: Copy) {
     into 'libs'
 }
 
-gradle.projectsEvaluated {
```

Comment:
This should not be removed since it disables the reverse proxy from your localhost to the phone for the running concrexit install. Also see my comment on the url util file.